modules/ballerina-native/src/test/java/org/ballerinalang/nativeimpl/connectors/JMSClientTest.java

```diff
@@ -51,22 +51,6 @@ public void testJMSClientConnectorWithoutValidInitialContextFactory() throws Bal
         BLangFunctions.invoke(bLangProgram, "jmsClientConnectorTest");
     }
 
-    @Test(description = "Test for jms client connector without valid message",
```

Comment:
@chanakaudaya I removed the test cases because now we are supporting empty messages. Therefore even if message data is not given we are sending empty messages. Anyway I will add a test case to cover the new implementation.